docs/plugins/client-crud-plugin.md

```diff
@@ -13,9 +13,15 @@ The plugin also creates your CRUD resolvers to use with your GraphQL schema.
 yarn add @graphback/codegen-schema
 ```
 
+To execute this plugin on-demand, you will use the [Graphback CLI](../graphback-cli). Install it as a devDependency in your project.
+
+```bash
+yarn add -D graphback-cli
+```
+
 ## Usage
 
-To use the plugin, add it to the `plugins` array in [`buildGraphbackAPI`](../api/build-graphback-api#plugins).
+You will use the [`graphback generate`](../graphback-cli#generate) CLI command to execute plugins on-demand.
```

Comment:
Feels weird recommending a completely different plugin execution flow just for one plugin.